src/app/artwork/[artworkId]/page.tsx

```diff
@@ -0,0 +1,56 @@
+import { NextPage } from "next";
+import Link from "next/link";
+
+
+interface PageProps {
+  params: {
+    artworkId: string;
+  };
+}
+
+const artworks = [
+  {"id": 1, "name": "Teruhito Sekine", "votes": 12, "url":"/gallery/artwork-1.png" },
+  {"id": 2, "name": "Teruhito Sekine", "votes": 15, "url":"/gallery/artwork-2.png"},
+  {"id": 3, "name": "Teruhito Sekine", "votes": 2,  "url":"/gallery/artwork-3.png" },
+  {"id": 4, "name": "Teruhito Sekine", "votes": 23, "url":"/gallery/artwork-4.png" },
+  {"id": 5, "name": "Teruhito Sekine", "votes": 12, "url":"/gallery/artwork-5.png" },
+  {"id": 6, "name": "Teruhito Sekine", "votes": 15, "url":"/gallery/artwork-6.png" },
+  {"id": 7, "name": "Teruhito Sekine", "votes": 2, "url":"/gallery/artwork-7.png" }
+];
+
+function getArtworkById(id:string) {
```

Comment:
I don't see a need for the `artworks` variable here or `getArtworkById`.

If you import artworks, you can just grab the desired artwork with `find`:

```
import { artworks } from "../../../../mock/artworks";

...

const artwork = artworks.find(artwork => artwork.id === Number(id));
```

I see the potential optimization using modulus here; however, I don't think the gain is enough for the cost of space and increased complexity of adding an additional artworks array.